crates/sui-benchmark/src/workloads/workload_configuration.rs

```diff
@@ -125,6 +132,14 @@ impl WorkloadConfiguration {
         let delegate_num_workers = (delegate_weight_ratio * num_workers as f32).ceil() as u64;
         let delegate_max_ops = delegate_qps * in_flight_ratio;
 
+        let batch_payment_weight_ratio = batch_payment_weight as f32
+            / (split_shared_counter_weight + split_transfer_object_weight + split_delegation_weight)
```

Comment:
denominator should include `batch_payment_weight`. Maybe make separate sum weight variable at the start of the function, so we won't forget the next time

Reply:
Derp, thanks for catching this

Reply:
I'm hoping we can refactor this code so each new workload type gets its weight etc computed automatically--do you think that is possible?

Reply:
yes, for sure. Overall it feels like there's a lot of unnecessary stuff we need to do each time we add a new workflow